Sources/NFCPassportReader/PassportReader.swift

```diff
@@ -12,11 +12,17 @@ import Foundation
 import UIKit
 import CoreNFC
 
+@available(iOS 15, *)
+public protocol PassportReaderDelegate: AnyObject {
+    func nfcTagDetected()
+}
+
 @available(iOS 15, *)
 public class PassportReader : NSObject {
     private typealias NFCCheckedContinuation = CheckedContinuation<NFCPassportModel, Error>
     private var nfcContinuation: NFCCheckedContinuation?
 
+    public weak var trackingDelegate: PassportReaderDelegate?
```

Comment:
```suggestion
    public weak var trackingDelegate: PassportReaderTrackingDelegate?
```

The "tracking" in the name is even already there  ☝️ 😄